cmake/Findjansson.cmake

```diff
@@ -0,0 +1,49 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+# Findjansson.cmake
```

Comment:
Since `libjansson=2.7` this Find module is not needed anymore because they provide a config. Could we make note of this in a comment somewhere? That way someone can easily make the call to remove this file in the future when we decide not to support `libjansson<2.7`.

Reply:
i don't understand if i don't have the Findjansson file cmake won't find the library.

Reply:
With a new enough version of `libjansson`, CMake should find the Config file that was installed along with its development headers. It moves the responsibility of setting up the dependency correctly to the dependency itself, instead of putting the burden on us.

However, I have seen a case before where a project correctly `install()`ed their config file, but it was missing from the packaging step. If this is the case here, it's a bug and we should probably report it to the project.

Reply:
is that this file?  /usr/lib64/pkgconfig/jansson.pc

Reply:
It should have the name janssonConfig.cmake or jansson-config.cmake. An example from my system is `/usr/lib64/cmake/c-ares-config.cmake`.

It looks like `janssonConfig.cmake` is only packaged on Fedora and Open Mandriva. That renders it effectively useless for any project that also wants to build on Ubuntu.

Reply:
BTW, CMake has integration with pkgconfig. I tried it out with when I added the lua plugin, but there was something unexpected I missed that @cmcfarlen fixed. So, we'll want to review that if we decide to use the pkgconfig integration.

Reply:
Opened akheron/jansson#668.